### ISSUE TITLE
Using ueberzugpp for image previews makes me exit my hyprland session or completely turn off my system.

### ISSUE THREAD

**Issue by jlumi382** [jstkdng/ueberzugpp#199]
After installing ueberzugpp from the aur, running `ueberzugpp layer` in my terminal (terminal emulator: `alacritty`) gave me this error `unable to load "/usr/lib/vips-modules-8.14/vips-openslide.so" -- libopenslide.so.0: cannot open shared object file`. I fixed it by installing libopenslide with `sudo pacman -S openslide`. this completely got rid of the error and ueberzugg would run without an error. However I never really got it to work, in fact it makes my laptop crash whenever I try to use ueberzugpp with "lobster -i" and image previews for "yazi". My laptop would just freeze in the middle and there would be no image preview as well. It would either eventually make me exit my hyprland session or completely turn off my system.

**Comment by jstkdng** [comment]
That is quite the predicament. Can you check if it also happens under X11 or other backend rather than `hyperland`?

**Comment by jlumi382** [comment]
Hello, i messed around with my drivers, I believe I used to have mesa, rocm-opencl-runtime, and vulkan-radeon installed (I don't really know what the problem is so I was just guessing and did some random guesses... I don't really know what I'm doing sorry 😭). These were the packages I installed a few hours ago `mesa lib32-mesa vulkan-radeon xf86-video-amdgpu libva-mesa-driver lib32-libva-mesa-driver mesa-vdpau lib32-mesa-vdpau vulkan-icd-loader lib32-vulkan-icd-loader opencl-clover-mesa opencl-rusticl-mesa lib32-opencl-clover-mesa lib32-opencl-rusticl-mesa rocm-opencl-runtime mesa-utils mesa mesa-vdpau mesa-demos`. And it now displays in yazi (either displays correctly or distorted), but with an error, it also displays a distorted image in lobster with an error.

![received_473457375263648.png](https://github.com/jstkdng/ueberzugpp/assets/156329665/bc73d48e-59c7-4a66-8cb2-be8fd5cab2e9)

![received_486664087378382.png](https://github.com/jstkdng/ueberzugpp/assets/156329665/30de4e9e-fa4c-40ff-9dcf-c3f634483c57)

**Comment by jlumi382** [comment]
Hello, i messed around with my drivers, I believe I used to have mesa, rocm-opencl-runtime, and vulkan-radeon installed (I don't really know what the problem is so I was just guessing and did some random guesses... I don't really know what I'm doing sorry 😭). These were the packages I installed a few hours ago `mesa lib32-mesa vulkan-radeon xf86-video-amdgpu libva-mesa-driver lib32-libva-mesa-driver mesa-vdpau lib32-mesa-vdpau vulkan-icd-loader lib32-vulkan-icd-loader opencl-clover-mesa opencl-rusticl-mesa lib32-opencl-clover-mesa lib32-opencl-rusticl-mesa rocm-opencl-runtime mesa-utils mesa mesa-vdpau mesa-demos`. And it now displays in yazi (either displays correctly or distorted), but with an error, it also displays a distorted image in lobster with an error.


Hello, I'm sorry for not knowing any better, I stuck with the opencl-rusticl-mesa drivers and removed the other ones which probably caused a conflict. I individually tried out rusticl, clover and rocm-opencl-runtime, my laptop was crashing because of rocm-opencl-runtime, and it worked well with the rusticl and clover drivers.

![received_473457375263648.png](https://github.com/jstkdng/ueberzugpp/assets/156329665/bc73d48e-59c7-4a66-8cb2-be8fd5cab2e9)

![received_486664087378382.png](https://github.com/jstkdng/ueberzugpp/assets/156329665/30de4e9e-fa4c-40ff-9dcf-c3f634483c57)
